src/styles/mixins/vertical-nav/_vertical-nav-navigation.scss

```diff
@@ -0,0 +1,374 @@
+@import '../../new-settings';
+@import '../mixins';
+
+$fd-list-navigation-item-height: 2.75rem !default;
+$fd-list-navigation-item-indicator-height: 1.625rem !default;
+$fd-list-navigation-item-indicator-spacing: 0.3125rem !default;
+$fd-list-navigation-item-text-spacing: 2.5rem !default;
+$fd-list-second-level-popover-position: 3.25rem !default;
+
+@mixin fd-vertical-nav-navigation($blockList: #{$fd-namespace}-list) {
+  .#{$blockList} {
+    .#{$blockList}__navigation-item-indicator {
+      @include fd-reset();
+
+      &::after {
+        content: '';
+        width: 0.1875rem;
+        display: inline-block;
+        position: absolute;
+        right: 0;
+        top: ($fd-list-navigation-item-height - $fd-list-navigation-item-indicator-height) * 0.5;
+        height: $fd-list-navigation-item-indicator-height;
+        background-color: var(--sapSelectedColor);
+        color: var(--sapSelectedColor);
+        border-radius: var(--sapField_BorderCornerRadius);
+        margin-right: $fd-list-navigation-item-indicator-spacing;
+      }
+
+      @include fd-rtl() {
+        &::after {
+          right: unset;
+          margin-right: 0;
+          left: 0;
+          margin-left: $fd-list-navigation-item-indicator-spacing;
+        }
+      }
+    }
+
+    &__navigation-item {
+      @include fd-reset();
+      @include fd-fiori-focus();
+      @include fd-set-paddings-x(0.5rem, 1rem);
+      @include _fd-vertical-nav-selected-color();
```

Comment:
why is this prefixed with `_`?

Reply:
I tried to achieve "private" mixins, i.e. that will be used only within this file.